_documentation/user/flags/jamming.md

```diff
@@ -5,4 +5,19 @@ type: Bad
 summary: Radar doesn't work. Can still see.
 ---
 
-...
+The Jamming flag causes static to block out radar, outside window viewing perspective is unaffected.
+
+## Tactics
+
+Jamming prevents any effective usage of radar, as it fills the radar with static.
+
+Countering the effects of Jamming requires good knowledge of the map and layout, along with effective moment patterns and positioning.
+If ricochet is enabled, one will not be able to notice shots coming from the rear, so avoiding areas where shots may ricochet into ones blind spot is recommended.
```

Comment:
Ricochet shots coming from your sides are also deadly. I'd opt towards saying to stay away from any area with a lot of ricos

Reply:
This could be generalized to just be about incoming shots from outside your field of view.

Reply:
Maybe the following would work?
> One will only be able to see shots coming from the outside window, so avoiding areas where shots may ricochet into or be directed towards ones blind spot is recommended.

Reply:
@Zehra I'm fine with that